### PR TITLE
add utility function to make reference attribute 

### PR DESCRIPTION
**Pull request by linkerzhang**
When creating function op, make reference attribute whose name is not the same as the attribute it refers.

### CODE REVIEW

File: onnx/defs/attr_proto_util.h [hunk 1]
```diff
@@ -8,16 +8,49 @@
...
+    const std::string& attr_name,
+    const std::vector<TensorProto>& values);
+AttributeProto MakeAttribute(
+    const std::string& attr_name,
+    const std::vector<GraphProto>& values);
 
-AttributeProto MakeRefAttribute(const std::string& attr_name, AttributeProto_AttributeType type);
+// Make a "reference" attribute for a node in a function body.
+// <attr_name> specifies the attribute name of both the function node and its
+// function body node. They're using same attribute name.
```
Comment (wschin) [inline onnx/defs/attr_proto_util.h]:
```suggestion
// function body node. They're using the same attribute name.
```

File: onnx/defs/attr_proto_util.h [hunk 2]
```diff
@@ -8,16 +8,49 @@
...
+// <type> specifies the attribute type.
+AttributeProto MakeRefAttribute(
+    const std::string& attr_name,
+    AttributeProto_AttributeType type);
+
+// Make a "reference" attribute for a node in a function body.
+// <attr_name> specifies the attribute name of the function body node.
+// <referred_attr_name> specifies the referred attribute name of the function
+// node. <type> specifies the attribute type.
```
Comment (wschin) [inline onnx/defs/attr_proto_util.h]:
```suggestion
// node.
// <type> specifies the attribute type.
```

File: onnx/defs/attr_proto_util.cc [hunk 3]
```diff
@@ -51,9 +51,16 @@ ADD_LIST_ATTR_IMPL(GraphProto, AttributeProto_AttributeType_GRAPHS, graphs)
 AttributeProto MakeRefAttribute(
     const std::string& attr_name,
     AttributeProto_AttributeType type) {
+  return MakeRefAttribute(attr_name, attr_name, type);
+}
+
+AttributeProto MakeRefAttribute(
```
Comment (wschin) [inline onnx/defs/attr_proto_util.cc]: Not sure how but I feel it's better to have at least one test.

**Comment by linkerzhang** [comment]
this is already covered by the PR #2575

**Pull request merged**